tgui/packages/tgui/interfaces/PortableChemMixer.js

```diff
@@ -42,9 +42,10 @@ export const PortableChemMixer = (props, context) => {
               <Button
                 key={chemical.id}
                 icon="tint"
-                width="150px"
-                lineHeight="21px"
+                width="425px"
```

Comment:
use `fluid` instead of width, since you want 100% width anyway

Reply:
yessir

Reply:
and you can remove mr={-1} in the box above too

Reply:
Is the change I made alright? I dont know if I commited a crime with my coding